### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by StephenAshmore** [mikegashler/waffles#12]
**Should we designate a version of waffles for release?**

Releases would be useful for those people who want a stable build that won't get changed every time we push to the repository. Having a designated release would make it easier to package into package managers, and perhaps for others to use for a specific application without worrying about a big change breaking everything.

**Comment by RadixSeven** [comment]
This sounds like a good idea. We've had success where I work using git tags to mark certain commits as release versions. 

    On Tuesday, April 12, 2016 2:05 PM, Stephen Ashmore <notifications@github.com> wrote:
 

 Should we designate a version of waffles for release?Releases would be useful for those people who want a stable build that won't get changed every time we push to the repository. Having a designated release would make it easier to package into package managers, and perhaps for others to use for a specific application without worrying about a big change breaking everything.—

**Comment by thelukester92** [comment]
I tagged the current commit as a stable release. Do we need to do anything else besides this?

**Comment by thelukester92** [comment]
(If you didn't know, this functionality is built into GitHub).

**Comment by StephenAshmore** [comment]
We were debating on what commit to tag as a stable release.

**Comment by thelukester92** [comment]
Ah, I misunderstood. Any reason not to arbitrarily choose the current commit?

**Comment by StephenAshmore** [comment]
Probably not. We have fixed quite a few small things, and training a neural network is pretty stable right now ( along with all the other learners ). Closing this one, we can continue release debate for a newer version of waffles eg #26 .

**Comment by RadixSeven** [comment]
Selecting a release as stable usually implies that extra care has been taken to fix all outstanding bug-level issues. Before a release there is commonly a feature-freeze on the to-be-released branch and known bugs are squashed. It is merely a convention, but a pretty strong one. 

    On Monday, October 24, 2016 9:33 PM, Stephen Ashmore <notifications@github.com> wrote:
 

 Probably not. We have fixed quite a few small things, and training a neural network is pretty stable right now ( along with all the other learners ). Closing this one, we can continue release debate for a newer version of waffles eg #26 .—
